src/index.js

```diff
@@ -38,10 +39,16 @@ export function isValid(value) {
  * E.g. "123456789" -> "XXXXX6789", "123-45-6789" -> "XXX-XX-6789".
  */
 
-export function mask(value) {
+function mask(value) {
   if (!isValid(value)) {
     throw new Error('Invalid Social Security Number');
   }
 
   return `${value.substr(0, value.length - 4).replace(/[\w]/g, 'X')}${value.substr(-4)}`;
 }
+
+/**
+ * Export `isvalid` and `mask`.
```

Comment:
Change to "Exports", like how we do on other projects? To avoid going out of sync in the future.